scout/export/variant.py

```diff
@@ -40,14 +42,46 @@ def export_variants(adapter, collaborator, document_id=None, case_id=None):
         # Add chromosome and position to prepare for sorting
         variants.append((chrom_int, variant_obj["position"], variant_obj))
 
-    # Sort varants based on position
+    # Sort variants based on position
     variants.sort(key=lambda x: (x[0], x[1]))
 
     for variant in variants:
         variant_obj = variant[2]
         yield variant_obj
 
 
+def export_managed_variants(
+    adapter: MongoAdapter,
+    institute: str = None,
+    build: str = None,
+    category: list = ["snv", "sv"],
+) -> ManagedVariant:
+    """Export managed variants, optionally for a given institute or variant category (["snv", "cancer_sv", ...])"""
+
+    # Store the variants in a list of tuples for sorting
+    variants = []
+
+    managed_variants = adapter.managed_variants(category=category, build=build, institute=institute)
+
+    for managed_variant_obj in managed_variants:
```

Comment:
What about sorting the results of the database query by chrom and position? You get 1, 11, 12 and so on? Could [this](https://stackoverflow.com/questions/57406820/mongodb-sort-gives-incorrect-result-with-alphanumeric-values) help in such case?

Reply:
I think that could work just as well, but not completely convinced it is better. I was about to say it would miss the index, as the index would be string based by default, not locale collated or whatever, but it seems we have no index on it so all good. That does appear a possible oversight to me for other uses! I guess the Big Thing for matching them is the churn of the incoming cases variants, plus the matching managed, but still. This collection is small, but the index would be really small and save some collection scans.

Reply:
The more I think about it, the more I like it. I'll just add the collated index. It will take little space, and drop a lot of old code that will run efficiently on the mongo server instead of on the web server.